<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddConnectionDialog</class>
 <widget class="QDialog" name="AddConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add new connection</string>
  </property>
  <property name="windowIcon">
   <iconset resource="myresources.qrc">
    <normaloff>:/icon/connections.png</normaloff>:/icon/connections.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableWidget" name="table_Computers">
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Scientists ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Scientists Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Computer ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Computer Name</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="input_Scientists_id">
     <property name="placeholderText">
      <string>Input Scientists ID ..</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="Input_Computers_id">
     <property name="placeholderText">
      <string>Input Computers ID ..</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_error">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="myresources.qrc"/>
 </resources>
 <connections/>
</ui>
